.github/workflows/argos.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
+  workflow_dispatch:
```

Comment:
Do we need to run it on demand?

Reply:
It's disabled right now. I want to be able to dispatch it manually though